src/Tools/Test Framework/Test Runner/src/TestMethodLine.Table.al

```diff
@@ -106,6 +106,18 @@ table 130450 "Test Method Line"
         {
             Editable = false;
         }
+        field(13; "Test Type"; Option)
```

Comment:
Do we need this field? Can't we do a lookup and show a value, we do not need to set it for every line. 
Also showing it on the codeunit is sufficient, test methods could skip the value. 
The value is read only so we do not need to store it in the database, unless there is something that Im missing.